pkg/cmd/release/download/download.go

```diff
@@ -46,13 +46,13 @@ func NewCmdDownload(f *cmdutil.Factory, runF func(*DownloadOptions) error) *cobr
 	cmd := &cobra.Command{
 		Use:   "download [<tag>]",
 		Short: "Download release assets",
-		Long: heredoc.Doc(`
+		Long: heredoc.Docf(`
 			Download assets from a GitHub release.
 
 			Without an explicit tag name argument, assets are downloaded from the
-			latest release in the project. In this case, "--pattern" or "--archive"
-                        is required.
-		`),
+			latest release in the project. In this case, %[1]s--pattern%[1]s or %[1]s--archive%[1]s
+			is required.
```

Comment:
Nice catch!

Reply:
Just to be sure, did you mean the space-to-tab restoration?